README.md

```diff
@@ -145,7 +145,7 @@ In summary, an Open MCT plugin is very simple: it's an initialization function w
 
 To be able to access our spacecraft objects from the tree, we first need to define a root. We will use the Open MCT API to define a new root object representing our spacecraft. 
 
-_[dictionary-plugin.js]()_
+_[dictionary-plugin.js](https://github.com/nasa/openmct-tutorial/blob/part-b-step-3/dictionary-plugin.js)_
```

Comment:
For some reason I italicized these, could you do me a favour and remove the `_`s? Thanks!